lib/appeals/responses/appeals.rb

```diff
@@ -7,8 +7,12 @@ class Appeals < Common::Base
       attribute :status, Integer
 
       def initialize(body, status)
-        self.body = body if json_format_is_valid?(body)
         self.status = status
+        if self.status == 404
+          self.body = { 'data' => [] }
```

Comment:
Is this what Caseflow actually returns when a user isn't found?

Reply:
no, the mock we have has this: 
```
:body:
  errors:
  - status: '404'
    title: Veteran not found
    detail: A veteran with that SSN was not found in our systems.
```

Based on the behavior in production / staging it seems like caseflow may actually return something more json schema compatible. We don't actually do much of anything with their responses, mearly just pass them through